src/components/Buttons/BasicButton/BasicButtonStyled.js

```diff
@@ -11,7 +11,6 @@ export const BasicButtonLayout = styled(Button)`
     line-height: 19.6px;
     text-align: center;
     padding: 12px 24px;
-    width: Hug (92px)px;
     border-radius: 4px;
     border: 1px 0px 0px 0px;
```

Comment:
Maybe here is dhould be just border-rop: 1px; ?

Reply:
this was changed.